sycl/test-e2e/InvokeSimd/Feature/popcnt_emu.cpp

```diff
@@ -126,7 +126,10 @@ int main(void) {
             } else {
               res = id % 2;
             }
-            sg.store(out_accessor.get_pointer() + offset, res);
+            sg.store(
+                out_accessor.get_multi_ptr<access::decorated::no>() +
```

Comment:
I think we want `access::decorated::yes` actually, does that work?

Reply:
For all files in this PR (and any future ESIMD test changes if you have them :) ) please, thanks